src/parse/read/directives.ts

```diff
@@ -24,7 +83,7 @@ function readExpression(parser: Parser, start: number, quoteMark: string|null) {
 			} else {
 				str += char;
 			}
-		} else if (/\s/.test(char)) {
+		} else if (/[\s\r\n\/>]/.test(char)) {
```

Comment:
`\s` should match the same character group as `[\s\r\n]`...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#Special_characters_meaning_in_regular_expressions
>Matches a single white space character, including space, tab, form feed, line feed and other Unicode spaces. Equivalent to [ \f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff].
> 
> For example, /\s\w*/ matches " bar" in "foo bar".
> 
> Matches a single white space character, including space, tab, form feed, line feed and other Unicode spaces. Equivalent to [ \f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff].For example, /\s\w*/ matches " bar" in "foo bar".

Reply:
ah whoops, missed this review before i hit merge, will account for these comments in a follow-up. sorry!